config.sh

```diff
@@ -325,6 +345,7 @@ function build_gdal {
         -DGDAL_USE_HEIF=OFF \
         -DGDAL_ENABLE_HEIF=OFF \
         -DGDAL_USE_ODBC=OFF \
+        -DGDAL_USE_LERC=ON \
```

Comment:
can't remember why but in lambgeo I use `-DGDAL_USE_LERC_INTERNAL=OFF` 
https://github.com/lambgeo/docker-lambda/blob/master/dockerfiles/Dockerfile.gdal3.5#L343

Reply:
can do although looking at the gdal docs I think it's off by default